src/famly_fetch/cli.py

```diff
@@ -1,13 +1,53 @@
+import re
+import time
+from dataclasses import dataclass
 from pathlib import Path
 
 import click
+import schedule
 
 from famly_fetch.downloader import FamlyDownloader
 
 
+@dataclass
+class ScheduleTime:
+    hour: int
+    minute: int
+
+
+class ScheduleTimeParamType(click.ParamType):
+    name = "schedule_time"
+
+    def convert(self, value, param, ctx):
+        if value is None:
+            return value
+        match = re.match(r"^(?:[01]\d|2[0-3]):[0-5]\d$", value)
+        if not match:
+            self.fail("Schedule time must be in HH:MM 24-hour format.", param, ctx)
+        hour, minute = map(int, value.split(":"))
+        return ScheduleTime(hour=hour, minute=minute)
+
+
 @click.command()
-@click.argument("email", envvar="FAMLY_EMAIL", type=str)
-@click.argument("password", envvar="FAMLY_PASSWORD", type=str)
+@click.option(
+    "--email",
+    envvar="FAMLY_EMAIL",
+    required=True,
+    prompt="Enter your famly.co email address",
+    help="Your famly.co email address, can be set via FAMLY_EMAIL env var",
+    metavar="EMAIL",
+    type=str,
+)
+@click.option(
+    "--password",
+    envvar="FAMLY_PASSWORD",
+    required=True,
+    prompt="Enter your famly.co password",
+    help="Your famly.co password, can be set via FAMLY_PASSWORD env var",
+    metavar="PASSWORD",
+    hide_input=True,
+    type=str,
+)
```

Comment:
change email and password to be options with prompt. So that user can input them when get prompted (password will be hidden), instead of providing them as command line argument. reducing the risk that password got leaked out.